spec/acceptance/beaker/git/basic_auth/basic_auth_checkout_http.rb

```diff
@@ -8,12 +8,14 @@
 http_server_script = 'basic_auth_http_daemon.rb'
 
 hosts.each do |host|
-  ruby = '/opt/puppet/bin/ruby' if host.is_pe? || 'ruby'
-  gem = '/opt/puppet/bin/gem' if host.is_pe? || 'gem'
+  ruby = host.is_pe? ? '/opt/puppet/bin/ruby' : 'ruby'
+  # ruby = '/opt/puppet/bin/ruby' if host.is_pe?
```

Comment:
Can we remove commented out code here and line 14

Reply:
Done